src/learning/lda.rs

```diff
@@ -0,0 +1,250 @@
+//! Latent Diriclhet Allocation Module
+//!
+//! Contains an implementation of Latent Diriclhet Allocation (LDA) using
+//! [Gibbs Sampling](http://psiexp.ss.uci.edu/research/papers/sciencetopics.pdf)
+//!
+//! LDA is typically used for textual analysis.  It assumes that a topic can be modeled
+//! as a distribution of words, and that a document can be modeled as a distribution of
+//! topics.  Thus, the output is these distributions on the input documents.
+//!
+//! Gibbs sampling is a Morkov Chain Monto Carlo algorithm that iteratively approximates
+//! the above distributions.
+//!
+//! This module doesn't use any training.  It uses unsupervised learning to estimate
```

Comment:
I find this statement a little confusing. I would argue that unsupervised learning algorithms still have a _training_ phase. However, during this training stage they are not given any target data.